src/System.ServiceModel.Syndication/tests/BasicScenarioTests.cs

```diff
@@ -51,16 +51,20 @@ public static void SyndicationFeed_Load_Write_RSS_Feed()
             try
             {
                 // *** SETUP *** \\\
-                XmlReader xmlr = XmlReader.Create(@"SimpleRssFeed.xml");
-                SyndicationFeed sf = SyndicationFeed.Load(xmlr);
-                Assert.True(sf != null);
+                SyndicationFeed sf;
+                using (XmlReader xmlr = XmlReader.Create(@"SimpleRssFeed.xml"))
+                {
+                    sf = SyndicationFeed.Load(xmlr);
+                    Assert.True(sf != null);
+                }
 
                 // *** EXECUTE *** \\
                 //Write the same feed that was read.
-                XmlWriter xmlw = XmlWriter.Create(path);
-                Rss20FeedFormatter atomFeed = new Rss20FeedFormatter(sf);
-                atomFeed.WriteTo(xmlw);
-                xmlw.Close();
+                using (XmlWriter xmlw = XmlWriter.Create(path))
+                {
+                    Rss20FeedFormatter atomFeed = new Rss20FeedFormatter(sf);
+                    atomFeed.WriteTo(xmlw);
+                }
 
```

Comment:
There's no verification that we round trip all data?

Reply:
Yeah, I noticed this problem. But this is not a new added test. I'm fixing the usage of `Close` here.